### PR TITLE
(feat) [SD-3531] Set a default place in user profile

### PR DESCRIPTION
**Pull request by sivakuna-aap**
Resolves [SD-3531](https://dev.sourcefabric.org/browse/SD-3531)

### CODE REVIEW

**Comment by mdhaman** [comment]
:+1:

File: client/app/scripts/superdesk-users/views/user-preferences.html [hunk 1]
```diff
@@ -23,7 +23,9 @@
             </li>
 
             <li class="fieldset terms-box">
-                <div class="legend">{{ :: preferences['dateline:located'].category }}</div>
+                <div class="legend">Article Defaults</div>
```
Comment (amagdas) [inline client/app/scripts/superdesk-users/views/user-preferences.html]: Missing translate.

File: client/app/scripts/superdesk-users/views/user-preferences.html [hunk 2]
```diff
@@ -23,7 +23,9 @@
             </li>
 
             <li class="fieldset terms-box">
-                <div class="legend">{{ :: preferences['dateline:located'].category }}</div>
+                <div class="legend">Article Defaults</div>
+
+                <div class="legend">Dateline</div>
```
Comment (amagdas) [inline client/app/scripts/superdesk-users/views/user-preferences.html]: Missing translate

**Comment by amagdas** [comment]
LGTM except the missing translates

File: server/apps/archive/common.py [hunk 3]
```diff
@@ -540,3 +499,40 @@ def convert_task_attributes_to_objectId(doc):
...
+            doc['dateline'] = {'date': current_date_time, 'source': ORGANIZATION_NAME_ABBREVIATION, 'located': None,
+                               'text': None}
+
+            if user and user.get('user_preferences', {}).get('dateline:located'):
+                located = user.get('user_preferences', {}).get('dateline:located', {}).get('located')
+                if located:
+                    doc['dateline']['located'] = located
+                    doc['dateline']['text'] = format_dateline_to_locmmmddsrc(located, dateline_ts)
+
+        if BYLINE not in doc and user and user.get(BYLINE):
```
Comment (petrjasek) [inline server/apps/archive/common.py]: just thinking - maybe some of those conditions could be simplified using `doc.setdefault` afterwards

**Comment by petrjasek** [comment]
:+1:

**Comment by amagdas** [comment]
:+1:

**Pull request merged**